bindings/netstandard/ElectionGuard/ElectionGuard.ElectionSetup/Guardian/GuardianStorageExtensions.cs

```diff
@@ -22,6 +21,7 @@ public static class GuardianStorageExtensions
     {
         return new(
             privateGuardianRecord.ElectionKeys,
+            privateGuardianRecord.CommitmentSeed,
```

Comment:
we also now have a commitment seed as part of the guardian record that is used to derive commitments. it's a secret value.